### DISCUSSION TITLE
the send button doesnt appear on my phone till i press symbols or space

### DISCUSSION THREAD

**Discussion by joeymeb** [flyerhq/flutter_chat_ui#600] [Q&A] [answered]
the send button doesnt appear on my phone till i press symbols or space also in my emulator but it works perfectly with the pc keyboard on the emulator

**Comment by demchenkoalex** [comment]
there is a mode for always visible one :)

**Comment by demchenkoalex** [comment, marked as answer]
v2 does not hide send button anymore https://github.com/flyerhq/flutter_chat_ui/issues/646
